packages/konsole-app/src/config.ts

```diff
@@ -6,7 +6,7 @@ import * as config from '@dxos/config';
 
 // Generated by webpack-version-file-plugin.
 // eslint-disable-next-line import/no-unresolved
-import buildInfo from '../version.json';
+import buildInfo from '../dist/version.json';
```

Comment:
i'm unsure if it's available before we build.

Reply:
As we discussed with Egor, this is some webpack magic, which nevertheless works for everyone